scripts/process_data.py

```diff
@@ -630,6 +642,68 @@ def get_matching_summary(num_intial_frames: int, num_matched_frames: int) -> str
     return f"[bold green]COLMAP found poses for {num_matched_frames / num_intial_frames * 100:.2f}% of the images."
 
 
+def polycam_to_json(
+    image_filenames: List[Path],
+    cameras_dir: Path,
+    output_dir: Path,
+    min_blur_score: float = 0.0,
+    crop_border_pixels: int = 0,
+) -> List[str]:
+    """Convert Polycam data into a nerfstudio dataset.
+
+    Args:
+        image_filenames: List of paths to the original images.
+        cameras_dir: Path to the polycam cameras directory.
+        output_dir: Path to the output directory.
+        min_blur_score: Minimum blur score to use an image. Images below this value will be skipped.
+        crop_border_pixels: Number of pixels to crop from the border of the image.
+
+    Returns:
+        Summary of the conversion.
+    """
+
+    frame_json = io.load_from_json(cameras_dir / f"{image_filenames[0].stem}.json")
+
+    data = {}
+    data["fl_x"] = frame_json["fx"]
+    data["fl_y"] = frame_json["fy"]
+    data["cx"] = frame_json["cx"] - crop_border_pixels / 2
+    data["cy"] = frame_json["cy"] - crop_border_pixels / 2
+    data["w"] = frame_json["width"] - crop_border_pixels
+    data["h"] = frame_json["height"] - crop_border_pixels
+    data["camera_model"] = CAMERA_MODELS["perspective"].value
+    data["orientation_override"] = "none"
+
+    frames = []
+    skipped_frames = 0
+    for i, image_filename in enumerate(image_filenames):
+        json_filename = cameras_dir / f"{image_filename.stem}.json"
+        frame_json = io.load_from_json(json_filename)
+        if "blur_score" in frame_json and frame_json["blur_score"] < min_blur_score:
+            skipped_frames += 1
+            continue
+        frame = {}
+        frame["file_path"] = f"./images/frame_{i+1:05d}{image_filename.suffix}"
+        frame["transform_matrix"] = [
+            [frame_json["t_20"], frame_json["t_21"], frame_json["t_22"], frame_json["t_23"]],
+            [frame_json["t_00"], frame_json["t_01"], frame_json["t_02"], frame_json["t_03"]],
+            [frame_json["t_10"], frame_json["t_11"], frame_json["t_12"], frame_json["t_13"]],
```

Comment:
It seems you have permuted the rows -- is this on purpose to achieve a particular target orientation? If so then looks fine to me, I would just leave a comment in the code since otherwise could be confusing